content/docs/hardware/general/case/_index.de.md

```diff
@@ -3,14 +3,224 @@ title: Gehäusedruck
+
+1. Der Import eines .stl files in eine slicer SW (wie z.B. Cura) ist der erste Schritt um ein 3D objekt zu drucken
+2. In der slicer SW wird der .stl file (oder mehrere files) möglichst so gelegt das es wenig Überhänge gibt
+{{< slider >}}
+{{< slider-image
+    src="OBS-MainCase-A02_lid_adfc-logo_gedreht.jpg"
+    alt="1. So soll es aussehen, aber die Drucklage ist anders " >}}
+{{< slider-image
+    src="OBS-MainCase-A02_lid_adfc-logo_Drucklage.jpg"
+    alt="2. Drucken mit dem Gesicht nach unten" >}}
+ {{< /slider >}}
+
+3. Einstellungen beim Vorbereiten
+{{< slider >}}
+  {{< slider-image
+    src="UM3_CuraUI_case_prepare.jpg"
+    alt="1. Cura view auf main case im Vorbereitungsmode" >}}
+  {{< slider-image
+    src="UM3_CuraUI_case_extruder.jpg"
+    alt="2. Einstellung auf PETG und 2 Extruder" >}}
+ {{< slider-image
+    src="UM3_CuraUI_case_settings.jpg"
+    alt="3. Settings für Druckdichte (Infill), Support, Düsendurchmesser" >}}
+{{< slider-image
+    src="UM3_CuraUI_case_slice.jpg"
+    alt="4. starte slicing" >}}
+ {{< /slider >}}
+
+4. Slicen starten: Cura erstellt nun einen file im format .gcode, der alle Befehle für den speziellen Drucker enthält (z.B. Druckdichte, Material, etc.). 
+{{< slider >}}
+{{< slider-image
+    src="UM3_CuraUI_case_slice_information.jpg"
+    alt="1. Nachdem alle Einstellungen erfolgt sind, wird das slicing gestartet " >}}
+{{< slider-image
+    src="UM3_CuraUI_case_slice_preview.jpg"
+    alt="2. Slicing information gibt es preview, mit den beiden Schiebern unten und rechts ist jede einzelne Bahn die gedruckt wird sichtbar" >}}
+ {{< /slider >}}
+
+5. Übertrag auf den 3D Drucker: Nachdem mein verwendeter Drucker in einem anderen Gebäude steht, speichere ich den .gcode file auf einem USB stick und stecke diesen an den Drucker. 
+6. Im Ultimaker 3 Menu wird dann der File für den Druck ausgewählt und der Druck gestartet.
+
+### Sensor gehäuse - Main case
+{{< imgproc "OBS_Gehäuse_mit_Deckel_und_text.jpg" Resize 600x >}} 
+Sensorgehäuse mit Deckel und Montage{{< /imgproc >}}
+Die Sensoren und Elektronik wird im Sensorgehäuse eingebaut. Empfohlen wird hier das Gehäuse mit dem OBS logo zu drucken, einmal aus Respekt zur OBS Community aber auch um damit zu werden. 5 Teile sind zum Drucken
+1. Für das Gehäuse mit dem OBS Logo habe ich folgende 2 files verwendet:   
+[OBS-MainCase-B-001a_MainCase_with_0.4mm_OBS-logo.stl](https://github.com/openbikesensor/OpenBikeSensor3dPrintableCase/blob/master/MainCase/VerticalCase_JSN-AJ/OBS-MainCase-B-001a_MainCase_with_0.4mm_OBS-logo.stl)   
+und [OBS-MainCase-B-001b_inner_logo_part.stl](https://github.com/openbikesensor/OpenBikeSensor3dPrintableCase/blob/master/MainCase/VerticalCase_JSN-AJ/OBS-MainCase-B-001b_inner_logo_part.stl)
+2. Das zweite Teil ist der Gehäusedeckel (lid), diesen gibt es ohne text, oder mit dem OBS logo, oder mit ADFC logo wie ich ihn gedruckt habe. Dazu braucht man 2 files:
+liegen auf ADFC page
```

Comment:
> … liegen auf ADFC page

Ist das öffentlich zugänglich? Dann bitte einen Link angeben.

Reply:
Das ist ein wichtiger Punkt, denn ich eh ansprechen wollte. Ich denke wir sollten spezifische Designs durchaus in einem eigenen Folder auf Github ablegen können, grade für ADFCler wäre es gut auch auf GitHub das entsprechende Gehäuse und Deckel zu finden, incl. Versionskontrolle. Derzeit sind die Dateien im ADFC Aktiven Bereich zu finden.  Wahrscheinlich gibt es noch einige andere Gruppierungen die mehr als ein Gehäuse drucken und ihr Logo auch zentral ablegen würden.

Reply:
ADFC page verlinkt (ist nur für Aktive zugänglich), doch nach wie vor fände ich als Ablageort Github besser für kundenspezifische Druckelemente wie das ADFC Logo, als spezielle webpages.